lrauv_ignition_plugins/worlds/portuguese_ledge.py

```diff
@@ -0,0 +1,17 @@
+# Takes in a template and generates an SDF file.
+
+import em
+import os
+import sys
+
+def main():
+    in_file = 'portuguese_ledge.sdf.em'
+    out_file = 'portuguese_ledge.sdf'
+    template = open(in_file).read()
```

Comment:
@chapulina consider: 
```suggestion
    with open(in_file) as infile:
        template in_file = .read()
```
to force the file to close.